great_expectations/expectations/expectation.py

```diff
@@ -329,7 +337,11 @@ class Expectation(metaclass=MetaExpectation):
         2. Data Docs rendering methods decorated with the @renderer decorator. See the
     """
 
-    version: ClassVar = ge_version
+    __metaclass__ = MetaExpectation
```

Comment:
Alleviates the errors I was seeing with metaclass conflicts but no longer registers as intended